nginx/Dockerfile

```diff
@@ -1,12 +1,19 @@
 FROM        phusion/baseimage:latest
```

Comment:
Is `phusion/baseimage:latest` still the best base image for the nginx container?

Reply:
Yes, it's working fine and pretty lightweight. I will revisit it soon though.